### ISSUE TITLE
 Serverumzug EVOLUTION - wie funktioniert das? (bitte einfache Erklärung)

### ISSUE THREAD

**Issue by flo001** [evolution-cms/evolution#897]
Hallo zusammen,

ich möchte eine komplette ModX - EVOLUTION (!!! nicht Revolution!)- auf einen anderen Server umziehen (aktuellste Evolution-Version: 1.2.1). Ich habe hierfür die komplette Webseite auf den neuen Server via FTP hochgeladen. Ich weiß, dass es hiermit noch nicht getan ist - es funktioniert ja auch noch nichts - nur weiß ich auch nicht, was ich alles ändern / einstellen muss und an welche Dateien ich konkret ran muss!

Auch ist zu erwähnen, aber das das wisst ihr bestimmt, dass ich derzeit nicht ins Backend (= Manager) reinkomme, da die base_URL ja nicht (mehr) stimmt! Ich habe allerdings auch nicht gefunden, wo ich die ändern kann. Ich habe diesen Punkt zwar in einem anderen ModX-Evolution-Backend gefunden und wenn man ins Backend kommt, kann man die ändern, aber da ich bei dieser betreffenden Installation ja wiegesagt nicht mehr ins Backend komme, kann ich es dort auch nicht ändern.

Vielleicht kann mir das jemand einfach und verständlich erklären. Denn leider findet man genügend Dokumentation zum Serverumzug von Revolution-Installationen, aber keine zu Evolution - und eine verständliche schon gleich gar nicht!

Anlehnend hierzu noch eine Frage:
Für mich kommen folgende 2 Szenarien infrage:
1.) Umzug der Installation auf eine Subdomain (z.B. www.subdomain.meine-webseite.de) oder
2.) in einen Unterordner / als Menüpunkt (z.B. wwww.meine-webseite.de/modx).
Frage: Ist dies überhaupt möglich? Dass man eine ModX-Installation auf eine Subdomain stellen kann, ist klar, das geht, aber ist die andere Variante auch möglich, dass ich meine Haupt-Domain habe und dann die Webseite "nur" auf einen Untermenüpunkt stelle? Oder braucht eine eigene, komplette Webseite auch immer eine komplette eigenständige Domain (bzw. Subdomain) für sich alleine?

Danke für eure Hilfen und dass ihr Licht ins Dunkel bringt.

Gruß in die Runde,
Flo

**Comment by dmi3yy** [comment]
Yes, you can transfer the site to a subdomain or to a subfolder, you can not take out the admin panel only to the subdomain, but you can change the folder manager any other. changing the folder name + path to it in the assets / cache / siteManager.php file

To transfer the site, you must copy all the files + copy the database and deploy it on the new hosting and then register the access to the database in manager / includes / config.inc.php
============
Ja, Sie können die Site in eine Subdomain oder in einen Unterordner übertragen. Sie können den Admin-Bereich nicht nur in die Subdomain verschieben, sondern Sie können den Ordner-Manager beliebig ändern. Ändern Sie den Ordnernamen und den Pfad dazu in der Datei assets / cache / siteManager.php

Um die Site zu übertragen, müssen Sie alle Dateien kopieren + die Datenbank kopieren und auf dem neuen Hosting bereitstellen und dann den Zugriff auf die Datenbank in manager / includes / config.inc.php registrieren

**Comment by Deesen** [comment]
Hat alles funktioniert?
